managers/tests/test_views.py

```diff
@@ -225,3 +229,89 @@ def test_delete_project_function_view_should_not_delete_project_on_non_admin_req
         response = self.client.post(self.url)
         self.assertEqual(response.status_code, 302)
         self.assertEqual(Project.objects.all().count(), 1)
+
+
+class ManageTaskActivitiesInProjectView(TestCase):
```

Comment:
Did you check if our scripts for running test starts this? You missed in this name "Test" and also you give to test class the same name as view name. Please change it :)

Reply:
Yes our scripts run this test too, however I wil change it ASAP... sory :(

Reply:
It works because it inherits from `TestCase`.